src/platforms/index.mdx

```diff
@@ -13,6 +13,17 @@ To report to Sentry you’ll need to use a language-specific SDK. The Sentry tea
 
 If your use-case is very specific, or not covered by Sentry, you'll find documentation on how to report events on your own using our [_SDK API_](https://develop.sentry.dev/sdk/overview/).
 
+<Note>
+
+Sentry SDKs are not officially supported nor recommended for use in third-party SDKs and libraries due to technical limitations:
```

Comment:
The reasons are explained below and are beyond technical limitations so simpler is better here:

```suggestion
Sentry SDKs are not officially supported nor recommended for use in third-party SDKs and libraries:
```